Framework/Algorithms/src/CalculateIqt.cpp

```diff
@@ -143,9 +143,8 @@ void CalculateIqt::exec() {
   m_resolutionIntegral = integration(rebin(resolution, rebinParams));
 
   resolution = fourierTransform(resolution, rebinParams);
-  if (enforceNormalization) {
-    resolution = divide(resolution, m_resolutionIntegral);
-  }
+  // Always normalise the resolution to unity
+  resolution = divide(resolution, m_resolutionIntegral);
```

Comment:
The enforceNormalization flag can determine when normalization occurs, instead of enabling it every time to avoid altering the results of the data when using the algorithm.